### ISSUE TITLE
Can someone please assist me with a problem?

### ISSUE THREAD

**Issue by PianoTutorials** [googlemaps/js-samples#1052]
Hello! My name is Niek and i'm new here, I found a website where i could see Google Earth in isometric view. I have an question/request for that. I will attach an image to show what i am trying to find out.
I am trying to get a viewing angle between the 90 degrees button. So a 45 degree twist to make an isometric view, please see the image.

It is for my university project, due in a month so no hurry, but i would like to add this view. I'm an Urban Design student.

Could anyone help me out please? Would be much appriciated! [https://developers.google.com/maps/documentation/javascript/examples/aerial-simple#maps_aerial_simple-typescript](url)

![problem with 90 degree viewing angle](https://user-images.githubusercontent.com/44372807/159665373-2c1bc6d5-b10d-4c40-8771-a7f4b6e8e7ad.jpg)

**Comment by jpoehnelt** [comment]
@PianoTutorials Thank you for opening this issue. 🙏
Please check out these other resources that might be applicable:

- [Support Console](https://cloud.google.com/support/) - If you have a support contract.
- [Discord](https://discord.gg/9UpV862xZ9) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag
- [Google Maps Platform issue trackers](https://developers.google.com/maps/support/#issue_tracker)


_This is an automated message, feel free to ignore._

**Comment by PianoTutorials** [comment]
Hmm it doesnt seem my added URL works... Let me try again:
https://developers.google.com/maps/documentation/javascript/examples/aerial-simple#maps_aerial_simple-typescript
